spec/abi.dd

```diff
@@ -379,6 +379,7 @@ $(GNAME Namechars):
 $(GRAMMAR
 $(GNAME LName):
     $(GLINK Number) $(GLINK Name)
+    $(GLINK Number) $(B __S) $(GLINK Number)    $(GREEN // fake parent symbols)
```

Comment:
are comments like this used anywhere else? Would think a separate `$(GNAME FakeParentName):` section would be more consistent if there are existing tools parsing this spec file.

Reply:
> are comments like this used anywhere else? Would think a separate $(GNAME FakeParentName): section would be more consistent if there are existing tools parsing this spec file.

Yes, there is, here is an example.

![image](https://user-images.githubusercontent.com/5050669/143598071-79efe977-8942-45ad-9b71-10a60462a11e.png)

I believe that a separate node is overkill, at least for that purpose for one branch. I think it is easy to skip those as they are marked specifically different with a green tag, although another rationale could come up